### DISCUSSION TITLE
Welcome to NEXUS---MINDUSTRY-MOD Discussions!

### DISCUSSION THREAD

**Discussion by Gamer123-games** [Gamer123-games/NEXUS---MINDUSTRY-MOD#5] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.
  * Always remember i suck 
  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Gamer123-games** [comment]
When someone talks say "Gamer123-games is bad game deveopment"

**Comment by GabyCostilla** [comment]
Hello, how can I collaborate in this project?
